### DISCUSSION TITLE
Application incorrectly rejected multiple times.

### DISCUSSION THREAD

**Discussion by skifli** [community/community#55196] [GitHub Education]
### Hi everyone,

Around a year ago I successfully got GitHub Student Pack on an old account before the verification system was updated. Even though I am using a school-verified email in the application, it still asks for more proof of my academic status. Now I wouldn't mind uploading proof of my status, except for the fact that my school doesn't give us any. In the end I used multiple pictures of my emails on Outlook and my SIMS Timetable. However, the automated system keeps on rejecting the application before it can get any further. It says I am not in the same country as my, except for the fact that I am. I am not using a VPN, and I checked my location on Google Maps and via my IP. Both place me within miles of my school at maximum, and most definitely in the same country.

I know this is not just a problem I am facing as other students to my school have had the same problem (@notjoshmiles). I ended up contacting GitHub support about this, but I got no response and then they decided to move to GitHub Community for Student Pack related inquires at that time, which was just great.

Anyway, I tried again today and I got rejected again for the exact same reasons. I was wondering if anyone could possible help me with this here, as GitHub decided to shut down the one direct way to contact them with issues related to the GitHub student pack.

![image](https://github.com/community/community/assets/121291719/e0b438e4-acf3-4446-9b69-49106ff5f960)

**Comment by samus-aran** [comment]
HI @skifli 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and I'll be happy to assist!

**Reply by skifli** [reply]
Hi, thanks for the (delayed) response. I ended up contacting GitHub Support using a different title for the issue, and it was resolved. Turned out that GitHub had our school marked out in a country around 5,000 miles away from where it actually is, but thankfully the support agent mentioned the country as one of the reasons why my application was rejected, so I saw it.
